src/modules/paymentProcessor/bridging/PP_Connext_Crosschain_v1.sol

```diff
@@ -0,0 +1,118 @@
+pragma solidity ^0.8.20;
+
+import {CrosschainBase_v1} from
+    "src/modules/paymentProcessor/bridging/abstracts/CrosschainBase_v1.sol";
+import {ICrossChainBase_v1} from
+    "src/modules/paymentProcessor/bridging/abstracts/ICrosschainBase_v1.sol";
+import {IERC20PaymentClientBase_v1} from
+    "@lm/interfaces/IERC20PaymentClientBase_v1.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {PP_Crosschain_v1} from
+    "src/modules/paymentProcessor/bridging/abstracts/PP_Crosschain_v1.sol";
+
+interface IEverclearSpoke {
```

Comment:
Please move these interfaces into separate files and put them into the interface folder